docs/admin/memory.rst

```diff
@@ -23,9 +23,16 @@ The right memory configuration depends on your workloads.
 Recommendation
 ==============
 
-A good starting point for the heap space is 25% of the system memory. However,
-it shouldn't be set below 1GB and not above 30.5GB, see the limits section
-below.
+A good starting point for the heap space is 25% of the system memory.
+
+Be careful when configuring this, as making the heap too large may lead to OutOfMemory errors,
+if the operating system cannot allocate memory for other needs, and on the other hand, making
+the heap too small may lead to OutOfMemory errors when new Java objects cannot be allocated
+inside the heap.
+
+In any case, it shouldn't be set below 1GB and not above 30.5GB,
+See the limits section below.
```

Comment:
I would like to ask a question about the very detail of the upper memory limit recommendation, but not within the conversation on this PR. I am still using it to spawn a dedicated ticket from. Thanks.